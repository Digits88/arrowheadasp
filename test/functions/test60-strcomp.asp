<!-- Test StrComp function.
-->

StrComp("cccc", "cccc") = <%= Strcomp("cccc", "cccc") %><br>
StrComp("cccc", "bbbb") = <%= Strcomp("cccc", "bbbb") %><br>
StrComp("cccc", "dddd") = <%= Strcomp("cccc", "dddd") %><br>
StrComp("aaaa", "AAAA") = <%= Strcomp("aaaa", "AAAA") %><br>
StrComp("aaaa", "AAAA", vbTextCompare) =
	<%= Strcomp("aaaa", "AAAA", vbTextCompare) %><br>
StrComp("aaaa", "AAAA", vbBinaryCompare) =
	<%= Strcomp("aaaa", "AAAA", vbBinaryCompare) %><br>
IsNull(StrComp(NULL, "aaa")) = <%= IsNull(StrComp(NULL, "aaa")) %><br>
IsNull(StrComp("aaa", NULL)) = <%= IsNull(StrComp("aaa", NULL)) %><br>
IsNull(StrComp(NULL, NULL)) = <%= IsNull(StrComp(NULL, NULL)) %><br>
