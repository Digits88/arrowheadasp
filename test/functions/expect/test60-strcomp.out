<!-- Test StrComp function.
-->

StrComp("cccc", "cccc") = 0<br>
StrComp("cccc", "bbbb") = 1<br>
StrComp("cccc", "dddd") = -1<br>
StrComp("aaaa", "AAAA") = 1<br>
StrComp("aaaa", "AAAA", vbTextCompare) =
	0<br>
StrComp("aaaa", "AAAA", vbBinaryCompare) =
	1<br>
IsNull(StrComp(NULL, "aaa")) = True<br>
IsNull(StrComp("aaa", NULL)) = True<br>
IsNull(StrComp(NULL, NULL)) = True<br>
